<launch>
  <!-- joy node -->
    <node respawn="true" pkg="joy" type="joy_node" name="joystick_node" >
      <param name="dev" type="string" value="/dev/input/js0" />
      <param name="autorepeat_rate" value="80" />
      <!-- <param name="coalesce_rate" value="2" /> -->
      <param name="deadzone" value="0.12" />
      <param name="output" value="screen" />
    </node>

  <!-- Axes -->
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="0" type="int"/>
    <param name="scale_linear" value="2" type="double"/>
    <param name="scale_angular" value="2" type="double"/>
    <node pkg="usu-ares-manipulation-2019" type="usu-ares-manipulation-2019_node" name="arm_control_node" output="screen"/>

  <!-- ROS Serial -->
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
      <param name="port" value="/dev/ttyACM0"/>
      <param name="baud" value="115200"/>
    </node>
</launch>
